### PR TITLE
Enhancement: ability to adjust rendering interval 

### PR DESCRIPTION
**Pull request by lvybriage**
Fix issue #1106

### CODE REVIEW

File: src/webui/package.json [hunk 1]
```diff
@@ -9,11 +9,12 @@
     "copy-to-clipboard": "^3.0.8",
     "echarts": "^4.1.0",
     "echarts-for-react": "^2.0.14",
-    "react": "^16.4.2",
-    "react-dom": "^16.4.2",
+    "react": "^16.7.0-alpha.2",
+    "react-dom": "^16.7.0-alpha.2",
```
Comment (chicm-ms) [inline src/webui/package.json]: Is this upgrade necessary?  since they are alpha versions.

Comment (lvybriage) [reply src/webui/package.json]: I've forgot why change it. I'll test with before package.json again.

Comment (lvybriage) [reply src/webui/package.json]: Had changed back.

Comment (lvybriage) [reply src/webui/package.json]: If don't use react alpha version, will met the problem "_react.default.useContext is not a function". I haven't found a better solution for it.

Comment (chicm) [reply src/webui/package.json]: ok

Comment (chicm-ms) [reply src/webui/package.json]: ok

**Pull request merged**